src/constants/events.ts

```diff
@@ -7,4 +7,5 @@ export enum Events {
   TRANSACTION_TRIGGERED = 'TRANSACTION_TRIGGERED',
   SIGNING_TRIGGERED = 'SIGNING_TRIGGERED',
   CLICK = 'CLICK',
+  CUSTOM_EVENT = 'CUSTOM_EVENT',
```

Comment:
Should this just be CUSTOM? Just thinking out loud, no strong preference.


```suggestion
  CUSTOM = 'CUSTOM',
```

Reply:
Totally lol